src/components/forms/UpdatePassword.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import 'tailwindcss/tailwind.css';
+import { setCurrentPassword,setConfirmPassword,setError,setLoading,setNewPassword } from "../../redux/reducers/userSlice";
+import {AiOutlineEyeInvisible,AiOutlineEye} from 'react-icons/ai'
+import { updatePassword } from "../../redux/reducers/userSlice";
+
+
+export const UpdatePassword=()=>{
+      const [currentPasswordError,setCurrentPasswordError]=useState('')
+      const [newPasswordError,setNewPasswordError]=useState('')
+      const [confirmPasswordError,setConfirmPasswordError]=useState('')
+      let   [currentOpenEye,setOpenEye]=useState(false)
+      let   [ newOpenEye,setNewOpenEye]=useState(false)
+      let   [confirmOpenEye,setConfirmOpenEye]=useState(false)
+      const dispatch=useDispatch();
+      const navigate=useNavigate();
+      const currentToggle=()=>{
+        setOpenEye(!currentOpenEye)
+      }
+      const newToggle=()=>{
+        setNewOpenEye(!newOpenEye)
+      }
+      const confirmToggle=()=>{
+        setConfirmOpenEye(!confirmOpenEye)
+      }
+
+      const passwordValidations=()=>{
+       let  isPwdValid=true;
+       if(newPassword !==confirmPassword){
+        setConfirmPasswordError("password don't march!")
+        isPwdValid=false;
```

Comment:
Please can we use selected libraries instead?  coz this is too much

Reply:
thanks for the feedbacks ttl